### PR TITLE
Prepare release v2.0

### LINKED ISSUE #97: Bump the registrar to v2
https://github.com/kubernetes-csi/node-driver-registrar/pull/96 introduces a breaking change. The next release should be v2.0.

### PR DESCRIPTION
**Pull request by jsafrane**
/kind cleanup

**What this PR does / why we need it**:
* Bump client-go to the latest RC
* Move changelogs to a separate dir
* Add changelog for v2

**Which issue(s) this PR fixes**:

Fixes #97

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

cc @msau42, PTAL

### CODE REVIEW

File: CHANGELOG/CHANGELOG-2.0.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+[Documentation](https://kubernetes-csi.github.io/docs/)
+# Changelog since v1.3.0
+
+## Changes by Kind
+
+### Feature
+
+- The Dockerfile must accept a "binary" build argument. ([#84](https://github.com/kubernetes-csi/node-driver-registrar/pull/84), [@pohly](https://github.com/pohly))
+
+### Uncategorized
```
[the remark this answers is not in the document]
Comment (jsafrane) [reply CHANGELOG/CHANGELOG-2.0.md:12]: Added

**Comment by jsafrane** [comment]
Fixed gcr.io registry, but I can't pull anything from it.
```
$ docker pull k8s.gcr.io/sig-storage/csi-attacher
Using default tag: latest
Error response from daemon: manifest for k8s.gcr.io/sig-storage/csi-attacher:latest not found: manifest unknown: Failed to fetch "latest" from request "/v2/sig-storage/csi-attacher/manifests/latest
```

(same for `k8s.gcr.io/sig-storage/node-driver-registrar`).

**Comment by msau42** [comment]
we only publish official release images. These are the tags available: https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml

Unofficial canary and rc images are available at gcr.io/k8s-staging-sig-storage but those images get auto deleted after a few weeks

**Comment by msau42** [comment]
Does https://github.com/kubernetes-csi/node-driver-registrar/pull/61 need a release note?

**Comment by jsafrane** [comment]
> Does #61 need a release note?

Added

**Pull request merged**